### PR TITLE
[Tizen] Add ble peripheral functionality

### PR DESCRIPTION
**Pull request by wootak-jung**
#### Problem
The ble peripheral role is not implemented.

#### Change overview
Add ble peripheral functionality in Tizen platform.

#### Testing
lighting-app example.

### CODE REVIEW

File: src/platform/Tizen/BLEManagerImpl.cpp [hunk 1]
```diff
@@ -37,35 +40,694 @@ namespace Internal {
...
 }
 
 CHIP_ERROR BLEManagerImpl::_SetAdvertisingEnabled(bool val)
 {
-    return CHIP_ERROR_NOT_IMPLEMENTED;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    if (mFlags.Has(Flags::kAdvertisingEnabled) != val)
+    {
+        mFlags.Set(Flags::kAdvertisingEnabled, val);
```
Comment (bzbarsky-apple) [inline src/platform/Tizen/BLEManagerImpl.cpp:664]: What is the purpose of this if block?  Is the ScheduleWork call meant to be inside it too?  If not, why not just set the flag unconditionally?

**Review by bzbarsky-apple — APPROVED** [review]
I did not do a careful review of the various bluetooth bits....

File: src/platform/Tizen/BLEManagerImpl.cpp (hunk 1 above)
Comment (wootak-jung) [reply src/platform/Tizen/BLEManagerImpl.cpp]: Thank you for your comment. I removed if block.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@saurabhst @andy31415 @Damian-Nordic ?

**Review by saurabhst — APPROVED** [review]

**Review by jepenven-silabs — APPROVED** [review]

**Review by msandstedt — APPROVED** [review]

**Pull request merged**